fixtures/builder-dependency-cts-config/electron-builder.config.cts

```diff
@@ -0,0 +1,9 @@
+interface Config {
+  productName: string;
+}
+
+const config: Config = {
+  productName: 'builder-dependency-cts-config',
+};
+
+export default config;
```

Comment:
this file is cjs, so I think need to use `module.exports` instead of `export default`

```
module.exports = config;
```